src/applications/vaos/utils/error.js

```diff
@@ -1,13 +1,25 @@
 import * as Sentry from '@sentry/browser';
+import { recordVaosError } from './events';
+
+export function captureError(err, skipRecordEvent) {
+  let eventErrorKey;
 
-export function captureError(err) {
   if (err instanceof Error) {
     Sentry.captureException(err);
+    eventErrorKey = err.message;
   } else {
     Sentry.withScope(scope => {
       scope.setExtra('error', err);
+      const message = `vaos_server_error${
+        err?.[0]?.title ? `: ${err?.[0]?.title}` : ''
```

Comment:
I think changing it to `err?.errors?.[0]?.title` would get rid of the undefineds in Sentry. 

Also, I think you could do `err?.errors?.[0]?.title || ''` and it's a little more concise.